docs/sources/datasources/pyroscope/_index.md

```diff
@@ -37,13 +41,13 @@ In this case, the screenshot shows memory profiles alongside panels for logs and
 
 ## Visualize traces and profiles data using Traces to profiles
 
-You can link profile and tracing data using your Pyroscope data source with the Tempo data source.
+You can link profile and tracing data using your Pyroscope data source with the Tempo data source. To learn more about how profiles and tracing can work together, refer to [Profiling and tracing synergies](./profiling-tracing-intro).
 
 Combined traces and profiles let you see granular line-level detail when available for a trace span. This allows you pinpoint the exact function that's causing a bottleneck in your application as well as a specific request.
 
 ![trace-profiler-view](https://grafana.com/static/img/pyroscope/pyroscope-trace-profiler-view-2023-11-30.png)
 
-For more information, refer to the [Traces to profile section][configure-tempo-data-source] of the Tempo data source documentation.
+For more information, refer to the [Traces to profile section][configure-tempo-data-source] and [Link tracing and profiling with span profiles](/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/trace-span-profiles/).
```

Comment:
nit: for consistency with Writers' Toolkit guidance https://grafana.com/docs/writers-toolkit/write/links/

```suggestion
For more information, refer to the [Traces to profile section][configure-tempo-data-source] and [Link tracing and profiling with span profiles](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/trace-span-profiles/).
```